client/my-sites/people/followers-list/index.jsx

```diff
@@ -162,7 +168,29 @@ const Followers = localize(
 				return <EmptyContent title={ emptyTitle } />;
 			}
 
-			let headerText = this.props.label;
+			let headerText;
+			if ( this.props.totalFollowers ) {
+				headerText = this.props.translate(
+					'You have %(number)d follower',
+					'You have %(number)d followers',
+					{
+						args: { number: this.props.totalFollowers },
+						count: this.props.totalUsers,
+					}
+				);
+
+				if ( this.props.type === 'email' ) {
+					headerText = this.props.translate(
+						'You have %(number)d email follower',
+						'You have %(number)d email followers',
+						{
+							args: { number: this.props.totalFollowers },
+							count: this.props.totalUsers,
```

Comment:
Same thing here; I think the `count` should match the `number`.

Reply:
Ooops, good point, fixed 👍